Local War/Assets/Resources/Script/Buff/Buff.cs

```diff
@@ -4,7 +4,8 @@
 
 public class Buff : MonoBehaviour {
 
-    protected string name;
+    protected string bname;
```

Comment:
Why did we change this name

Reply:
The unity generate an error that says the "name" actually override a built-in variable. So I have to change the name into bname, I will double-check if it will still generate the error.

Reply:
OK